recipes/napari-nd-annotator/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "napari-nd-annotator" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-nD-annotator-{{ version }}.tar.gz
+  sha256: fdb55d27848e876e8f27dde7a1f6b6cc15177e8b2a1764ed21fe1242f62da43f
+  patches:
+    - imports.patch  # Fix opencv import
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - magicgui
+    - matplotlib-base
+    - napari ==0.4.15
```

Comment:
Can we relax this requirement a bit @bauerdavid? Maybe `>=0.4.15`?

Reply:
Ok! We can update the pinning once the new release is available, but for now I will merge!